README.md

```diff
@@ -23,37 +23,40 @@ MIGProfiler is featured for:
  Coming soon!
 
 ## Install 📦️
+
+### Install by PyPI
```

Comment:
```suggestion
### Prepare requirments by PyPI
```
"install" could be confusing for users since we need to git clone repo and build it thereafter.

Reply:
Is the `migperf` package published to PyPI?

Reply:
If no, please open a issue and assign me @YuanmingLeee